js/src/components/RaisedURLButton.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import RaisedDefaultButton from "./RaisedDefaultButton";
+import InfoIcon from "material-ui/svg-icons/action/info";
+
+const RaisedURLButton = ( props ) => {
+	const buttonProps = Object.assign( {}, props );
+	delete buttonProps.url;
+
+	return (
+		<RaisedDefaultButton { ...buttonProps } url={ props.url } />
```

Comment:
The url is in the props. I would do this:
`<RaisedDefaultButton { ...props } />`